### PR TITLE
Better readme.

### PR DESCRIPTION
**Pull request by hemanth**
//cc @indutny Is it ok?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,101 @@
...
+// Little endian
+new BN([1,2,3], 10, 'le').toString(16); // '30201'
+new BN([1,2,3,4], 10, 'le').toString(16); // '4030201'
+
+// bitLength
+new BN(0x123456).bitLength(); // 21
+new BN('123456789', 16).bitLength(); // 33;
+
+// iaddn
+new BN(-100).sign  // true
```
Comment (indutny) [inline README.md]: `;` for consistency? :)

File: README.md [hunk 2]
```diff
@@ -1,8 +1,101 @@
...
+new BN(100).sign   // false
+
+// isubn
+new BN(-100).isubn(200) // -300
+
+// add
+new BN(14).add(new BN(26)).toString(16); // '28'
```
Comment (indutny) [inline README.md]: Considering that many people will try it out in io.js/node.js repl. May be we could just write?

```javascript
new BN(14).add(new BN(26));  // <BN: 0x28>
```

**Comment by indutny** [comment]
Looks awesome! A big step forward!

**Comment by indutny** [comment]
Thank you!

File: README.md (hunk 2 above)
Comment (hemanth) [inline README.md]: You mean to say, avoid all the `toString` calls?

Comment (indutny) [inline README.md]: Yeah, it has nice output because of `.inspect` method.

Comment (hemanth) [inline README.md]: Cool, I'll remove all the `toString` calls and add their respective outputs.

Comment (indutny) [inline README.md]: Thank you!

**Comment by indutny** [comment]
Yay, few missing semicolons, but I can fix it myself. LGTM

**Comment by indutny** [comment]
Thank you!

**Comment by hemanth** [comment]
Thank you for `bn,js`! 

We would need an icon as well...?

**Comment by indutny** [comment]
Landed, thank you!

Regarding icons - would be great!

**Pull request closed without merging**